routers/web/org/teams.go

```diff
@@ -339,7 +339,6 @@ func SearchTeam(ctx *context.Context) {
 	}
 
 	opts := &organization.SearchTeamOptions{
-		UserID:      ctx.Doer.ID,
 		Keyword:     ctx.FormTrim("q"),
```

Comment:
```suggestion
		// UserID is not set because the router already requires the doer to be an org admin. Thus, we don't need to restrict to teams that the user belongs in
		Keyword:     ctx.FormTrim("q"),
```
So that we have a good chance to find an error early in case this precondition ever gets changed.

Reply:
Thanks!